src/PayU/Alu/CardToken.php

```diff
@@ -13,14 +13,26 @@ class CardToken
      */
     private $token;
 
+    /**
+     * @var string
+     */
+    private $cvv;
+
+    /**
+     * @var string
+     */
+    private $owner;
+
     /**
      * @param string $token
      * @param string $cvv
+     * @param string $owner
      */
-    public function __construct($token, $cvv = '')
```

Comment:
A token includes also the owner name.
AFAIK, sending CC_OWNER in ALU v3 and authorization.merchantToken.owner in payment v4 is of no real use.

Reply:
done, removed owner param